DigitalOffice/Tests/ProjectServiceTests/CommandsTests/CreateNewProjectCommandTests.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Net;
+using FluentValidation;
+using Moq;
+using FluentValidation.Results;
+using NUnit.Framework;
+using ProjectService.Commands;
+using ProjectService.Commands.Interfaces;
+using ProjectService.Database.Entities;
+using ProjectService.Controllers.RestRequest;
+using ProjectService.Repositories.Interfaces;
+
+namespace ProjectServiceUnitTests.CommandsTests
+{
+    class CreateNewProjectCommandTests
+    {
+        private Mock<ValidationResult> validatorResultMock;
+        private ICreateNewProjectCommand command;
+
+        [SetUp]
+        public void SetUp()
+        {
+            var validatorMock = new Mock<IValidator<NewProjectRequest>>();
+            var repositoryMock = new Mock<ICreateNewProjectRepository>();
+            validatorResultMock = new Mock<ValidationResult>();
+
+            validatorMock
+                .Setup(x => x.Validate(It.IsAny<IValidationContext>()))
+                .Returns(validatorResultMock.Object);
+
+            command = new CreateNewProjectCommand(repositoryMock.Object, validatorMock.Object);
+        }
+
+        [Test]
+        public void FailCreateNewProjectIncorrectProjectDataTests()
+        {
+            validatorResultMock
+                .Setup(x => x.IsValid)
+                .Returns(false);
+
+            var newProject = new NewProjectRequest
+            {
+                Name = "DigitalOffice",
+                DepartmentId = Guid.NewGuid(),
+                Description = "New project for Lanit-Tercom",
+                IsActive = true,
+                WorkersUsersIds = null,
+                ManagersUsersIds = null,
+                FilesIds = null
+            };
+
+            Assert.Throws<ValidationException>(() => command.Execute(newProject), "Project field validation error");
+        }
+
+        [Test]
+        public void SuccessfulCreateNewProjectTest()
+        {
+            validatorResultMock
+                .Setup(x => x.IsValid)
+                .Returns(true);
+
+            var newProject = new NewProjectRequest
+            {
+                Name = "DigitalOffice",
+                DepartmentId = Guid.NewGuid(),
+                Description = "New project for Lanit-Tercom",
+                IsActive = true,
+                WorkersUsersIds = null,
+                ManagersUsersIds = null,
+                FilesIds = null
+            };
+
+            command.Execute(newProject).StatusCode.Equals(HttpStatusCode.OK);
```

Comment:
`Assert.Equals(..)`?

Reply:
Here the check is carried out exactly on the corresponding status code, it is also possible through Assert, there is almost no difference

Reply:
Do not quite understand. If, for example, change the expected status code to HttpStatusCode.ExpectationFailed, test will be successful, but it should not be this way.

Reply:
Ok, i agree, fixed